### PR TITLE
[yoga] Update to 1.19.0

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates yoga to 1.19.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/yoga/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "yoga",
-  "version-string": "1.18.0",
-  "port-version": 2,
+  "version-string": "1.19.0",
```
Comment (PhoebeHui) [inline ports/yoga/vcpkg.json:3]:
```suggestion
  "version-semver": "1.19.0",
```

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Sorry for the spurious CI failure; the tensorflow port is leaving behind processes which are causing subsequent runs to fail on some machines. I disabled the affected machine and resubmitted.

**Pull request merged**